File: ios/IOSCalculatorModule.m
//
//  CalculatorModule.m
//  HomeAssigment
//
//  Created by Ylli Petrovci on 27.2.22.
//

#import <Foundation/Foundation.h>
#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>

@interface RCT_EXTERN_MODULE(IOSCalculatorModule, NSObject)

RCT_EXTERN_METHOD(open:(double *)firstNumber secondNumber:(double *)secondNumber callback:(RCTResponseSenderBlock *)successCallback)

@end

File: ios/IOSCalculatorModule.swift
//
//  CalculatorModule.swift
//  HomeAssigment
//
//  Created by Ylli Petrovci on 27.2.22.
//

import Foundation
import AVFoundation


@objc(IOSCalculatorModule)
class IOSCalculatorModule : NSObject {
  @objc func open(_ firstNumber: Double, secondNumber: Double, callback successCallback: RCTResponseSenderBlock){
    
    let result = firstNumber + secondNumber ;
    
    print("Two numbers firstNumber:"+String(format: "%.1f", result))
    
    successCallback([result])
  }
}
